Cargo.toml

```diff
@@ -49,34 +48,32 @@ git = "https://github.com/KZen-networks/class-groups"
 tag = "v0.1.4"
 optional = true
 
-[dependencies.rocket]
-version = "0.4.2"
-default-features = false
-optional = true
-
-
-
-[patch.crates-io]
-rust-gmp = { version = "0.5.0", features = ["serde_support"], git = "https://github.com/KZen-networks/rust-gmp" }
-
 [dev-dependencies]
-criterion = "0.2"
+criterion = "0.3"
+rust-crypto = "0.2"
 hex = "0.3.2"
-rocket_contrib = {version = "0.4.2",default-features = false,features = ["json"]}
-rocket = {version = "0.4.2", default-features = false}
-reqwest = {version = "0.9.5", default-features = false}
+rocket = "0.4.2"
+rocket_contrib = "0.4.2"
+reqwest = { version = "0.9", default-features = false }
 uuid = { version = "0.7", features = ["v4"] }
-rust-crypto = "^0.2"
 serde_json = "1.0"
+libsecp256k1 = "0.2.2"
```

Comment:
why is this dependency needed?
I don't see it consumed anywhere. 
Also `curv` uses `secp256k1` and not `libsecp256k1` and for a very good reason I believe - `secp256k1` uses C bindings for libsecp256k1 in C, which is by far the most battle tested library for secp256k1...

Reply:
This is for testing signature against another secp256k1 repo

Rust wrapper of C bindings for libsecp256k1 is __not__ reliable as you think, see https://github.com/rust-bitcoin/rust-secp256k1/issues/163

Reply:
Of course, I'm just saying __the rust wrapper__ is with problem

Reply:
Can you please point me to where this package is consumed?

Reply:
libp2p rust and substrate, enough?
https://github.com/libp2p/rust-libp2p
https://github.com/paritytech/substrate/

Reply:
Even it is not widely used, there is no hurt if you use it for testing against signature

Reply:
And why we don't use C bindings?
The answer is:
https://github.com/rust-bitcoin/rust-secp256k1/issues/163